APE1.rst

```diff
@@ -1,19 +1,22 @@
 APE Purpose and Process
 -----------------------
 
-author: Perry Greenfield
+author: Perry Greenfield, Lia Corrales, Thomas Robitaille, Erik Tollerud, Pey Lian Lim
```

Comment:
there was a long discussion about not changing the original metadata: 

https://github.com/astropy/astropy-APEs/pull/60#pullrequestreview-502468338